Dockerfile.cpu

```diff
@@ -1,6 +1,14 @@
 FROM ubuntu:16.04
 
+ENV LC_ALL=C.UTF-8
+ENV LANG=C.UTF-8
+ENV PATH /opt/conda/bin:$PATH
+ENV PYTHONHASHSEED 2157
+
+WORKDIR /stage
 EXPOSE 8000
+ENTRYPOINT ["/bin/sh", "-c"]
```

Comment:
I would use the default entrypoint unless you're using this for shell scripting.